config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+8y568Ee3rnhfXg9fA1GalwyHE35RjXh6QTKWMaHYwNMwKBKlfGEVId1uCc910C2eROdDsqbeMe+LRzztNV5RMsN0DiEPxf9zSVeyG410M6ltgQvl+B/JrddklMBRKdrEATSQ5nQITZ/dp9ENPYQkMmQF+15PeRONu3/2FVw4DKyoHVVF9G2s95VperTUEUZpu7gz5eUxKa5vk6K1r8bd659L/z3hV4H473ezKoasSnOBE0E/qubyy9iIEq5XGwvApR5UCb/aQ9kJBTNO23MOXtFHZStFFwb5sCPIsnLOFJM2tNxPEChUtlmZhwcvaSqXDu8fBLNeBWpapdJM723u7YEQLzxqY/ZwciUpV5j5gXhvzYe0Ym22kFvHosL+6Z5qGKP/Dz1Acf9HEFtOjGGc7NxwpdxLCYQFHm6/--dsAjZBWWnfvc/c1k--Iecfs1nfNsODaFl5nAIcBg==
```

Comment:
file này cũng ko cần đưa lên github đâu em nhé

Reply:
file này em đưa vào git ignore ở pull sau rồi ạ nhưng mà em dùng lệnh git rm --cached credentials.yml.enc cho file này nhưng mà nó báo lỗi fatal: pathspec 'credentials.yml.enc' did not match any files á chị